### PR TITLE
Use template literals over string concatenation

### PR DESCRIPTION
**Pull request by tomtomdotcom**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -106,10 +106,9 @@ function parseSource(source) {
             
             switch(typeof obj[key]) {
                 case "string": break;
-                case "object": check(obj[key], path + "." + key); break;
+                case "object": check(obj[key], `${path}."${key}`); break;
```
Comment (asztal) [inline index.js]: Looks like a rogue `"`?

File: index.js [hunk 2]
```diff
@@ -156,7 +155,7 @@ function compile(source, config) {
         for (let key in obj) {
             const value = obj[key];
             if (typeof value === "object") {
-                const nested = convert(obj[key], path + key + ".");
+                const nested = convert(obj[key], `${path} + ${key}.`);
```
Comment (asztal) [inline index.js]: Here too, it needs to be `${path}${key}.` :)

**Review by asztal — CHANGES_REQUESTED** [review]
All for it, just some minor changes to make 👌

**Comment by tomtomdotcom** [comment]
^ PR updated with issues.

**Pull request merged**